src/lib/commandBarConfigs/modelingCommandConfig.ts

```diff
@@ -742,7 +742,7 @@ export const modelingMachineCommandConfig: StateMachineCommandSetConfig<
   Fillet: {
     description: 'Fillet edge',
     icon: 'fillet3d',
-    status: 'development',
+    status: 'active',
```

Comment:
I think we can either do this or delete the line, should be the same

Reply:
For consitency with other configs I'd delete but it doesn't matter

Reply:
done.